src/mlpack/methods/ann/layer/isrlu_impl.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file isrlu_impl.hpp
+ * @author Pranav Reddy P
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_ISRLU_IMPL_HPP
+#define MLPACK_METHODS_ANN_LAYER_ISRLU_IMPL_HPP
+
+// In case it hasn't yet been included.
+#include "isrlu.hpp"
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+template<typename InputDataType, typename OutputDataType>
+ISRLU<InputDataType, OutputDataType>::ISRLU(
+    const double alpha) : alpha(alpha)
+{
+  // Nothing to do here.
```

Comment:
I think it would more sense to have an assert statement here that ensures alpha is between -1 and 1.